app/assets/stylesheets/legal_forms.scss

```diff
@@ -0,0 +1,7 @@
+// Place all the styles related to the LegalForms controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.datepicker.dropdown-menu table {
+  background-color: #FFFFFF;
+}
```

Comment:
See this little red symbol in GitHub? It indicates that you don't have a newline character at the end of your file.

Here's [an article on why this is important](https://robots.thoughtbot.com/no-newline-at-end-of-file).

You can configure your editor to insert this automatically for you. If you're using Sublime Text, the configuration option is: `ensure_newline_at_eof_on_save`